tests/integration/test_data/pip_multiple_packages/fetch_deps_sha256sums.json

```diff
@@ -2704,381 +2704,6 @@
   "pip/charset-normalizer-3.2.0.tar.gz": "sha256:3bb3d25a8e6c0aedd251753a79ae98a093c7e7b471faa3aa9a93a81431987ace",
   "pip/click-8.1.7.tar.gz": "sha256:ca9853ad459e787e2192211578cc907e7594e294c7ccc834310722b41b9ca6de",
   "pip/cryptography-41.0.3.tar.gz": "sha256:6d192741113ef5e30d89dcb5b956ef4e1578f304708701b8b73d38e3e1461f34",
-  "pip/cryptography-41.0.3/CHANGELOG.rst": "sha256:4601a7b95e2757b672e736720deada09ba2f0a82c76a9832729bf90c54c8090c",
```

Comment:
Yeah, we need to sit down and figure out if we need ^these JSON files at all, it often just pollutes the diff :( .